concent_api/utils/fields.py

```diff
@@ -0,0 +1,24 @@
+import base64
+
+from django.db import models
+
+
+class Base64Field(models.TextField):
```

Comment:
Does this field expect the user to assign strings that are already base-64 encoded or unencoded binary data? Or can it handle both? This should be documented here.

Reply:
:+1:

Reply:
How will it behave if you assign data that can't be decoded with `b64decode()` to the `_base64` field? Are there any validations or checks against that?

Reply:
So what about this one?